module/sound/SpeechRecognition/README.md

```diff
@@ -0,0 +1,16 @@
+SpeechRecognition
+=================
+
+## Description
+This module takes in English speech as a wav file and converts it into text. It uses vosk-api to decipher the audio into text.
```

Comment:
```suggestion

This module takes in English speech as binary data and converts it into text. It uses vosk-api to decipher the audio into text.
```
Newline for consistency with other readmes. This module takes binary audio data, it isn't specific to the WavReader module.